extra-network/clash/autobuild/defines

```diff
@@ -1,6 +1,8 @@
 PKGNAME=clash
+# TO-DO: package provide what-so-ever
 PKGSEC=net
 PKGDEP="glibc"
 BUILDDEP="go"
 PKGDES="A rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols"
```

Comment:
When using alignment, the first and second to last words is usually connected with "and".

Although not yet encountered, a little tip for composing package description is that you can try your best to describe an application in an objective and neutral language as much as possible, instead of just copy description from upstream or other distributions.